lib/client-assets.php

```diff
@@ -185,7 +192,11 @@ function gutenberg_register_scripts_and_styles() {
 	wp_add_inline_script(
 		'wp-dom',
 		gutenberg_get_script_polyfill( array(
-			'document.contains' => 'wp-polyfill-node-contains',
+			'document.contains'   => 'wp-polyfill-node-contains',
+			'"DOMRect" in this && ( function( DOMRect ) {' .
+				'try { return new DOMRect(); return true; }' .
```

Comment:
`return true;` is unreachable here. We don't want the `return` on the `new DOMRect();` statement.

Reply:
🤦‍♂️ fixing...